release/raydp_tests/raydp_benchmark.py

```diff
@@ -0,0 +1,220 @@
+import ray
+import ray.autoscaler.sdk
+import glob
+import csv
+import pandas as pd
+import numpy as np
+from pyspark.sql import SparkSession
+import raydp
+import os.path
+
+import time
+
+
+def load_dataset(spark, nbytes, npartitions, sort, base):
+    num_bytes_per_partition = nbytes // npartitions
+    filenames = []
+
+    @ray.remote
+    def load(i):
+        filename = os.path.join(base, "df-{}-{}-{}.parquet".format("sort" if sort else "groupby", num_bytes_per_partition, i))
+        print("Partition file", filename)
+        if not os.path.exists(filename):
+            if sort:
+                nrows = num_bytes_per_partition // 8
+                print("Allocating dataset with {} rows".format(nrows))
+                dataset = pd.DataFrame(np.random.randint(0, np.iinfo(np.int64).max, size=(nrows, 1), dtype=np.int64), columns=['a'])
+            else:
+                nrows = num_bytes_per_partition // (8 * 2)
+                print("Allocating dataset with {} rows".format(nrows))
+                dataset = pd.DataFrame(np.random.randint(0, 100, size=(nrows, 2), dtype=np.int64), columns=['a', 'b'])
+            print("Done allocating")
+            dataset.to_parquet(filename, flavor="spark")
+            print("Done writing to disk")
+        return filename
+
+    for i in range(npartitions):
+        filenames.append(load.remote(i))
+    ray.wait(filenames, num_returns=len(filenames))
+
+    return spark.read.parquet(os.path.join(base, "df-{}-{}-*.parquet".format("sort" if sort else "groupby", num_bytes_per_partition)))
+
+
+def trial(spark, nbytes, n_partitions, sort, generate_only, base):
+    df = load_dataset(spark, nbytes, n_partitions, sort, base)
+
+    if generate_only:
+        return
+
+    times = []
+    start = time.time()
+    for i in range(10):
+        print("Trial {} start".format(i))
+        trial_start = time.time()
+
+        if sort:
+            df.sort("a").head(10)
```

Comment:
@rkooo567 You were suggesting that we should write out the full output as Parquet files instead of bringing back a small sample back to the driver, right?

Reply:
Let's actually see Stephanie's script and follow the same pattern.